File: test/functions/ruby/echo.rb
def Main(req)
	puts "req:"
	puts req
	puts "args:"
	puts req.args
	puts "content:"
	puts req.content
	puts "body:"
	puts req.body
	puts "b:"
	puts req.b
	if req.b != nil
		puts "b.name:"
		puts req.b.name
	end
	STDOUT.flush
	return { "myname" => "hw:ruby:" + req.args["name"] }, { "status"=>201 }
end

File: test/functions/python/echo.py
def Main(req):
    print("Object:" + " ".join(dir(req)))
    print("Ct: " + req.content)
    print("Args: " + ("%r" % req.args))
    try:
        print(req.claims)
    except:
        print("no claims")
    try:
        print(req.body)
    except:
        print("no body")
    try:
        print("B:" + ("%r" % req.b) + " :".join(dir(req.b)))
        return {"name": req.b.name }, {"status": 201}
    except:
        print("no b")
        return {"name": req.args["name"] }, {"status": 201}
